spec/models/task_spec.rb

```diff
@@ -1,16 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  subject(:task) do
-    Task.new(title: "Short text", 
-             description: "Long text", 
-             due_date: Date.today + 1.day )
-  end
-  
+  subject(:project) { FactoryGirl.create(:project) }
```

Comment:
I think that `project` is not your subject here, you can declare it using `let`.